xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-default/src/main/java/org/xwiki/mentions/internal/MentionXDOMService.java

```diff
@@ -48,11 +49,13 @@
     /**
      * Count the number of mentions per reference.
      *
-     * @param mentions the list of mentions.
+     * @param mentions the list of mentions
+     * @param wikiReference the reference to the wiki in which the mentions occured
      * @return the map of anchors of mentions per reference. If the anchor is not specified, then the list must contain
```

Comment:
Might be nice to explain this a bit more. I see it's returning a map with a ref as the key. What is this ref corresponding to? (basically what is an "anchor"?).